trin-beacon/src/storage.rs

```diff
@@ -222,13 +223,12 @@ impl ContentStore for BeaconStorage {
         })?;
 
         match beacon_content_key {
-            BeaconContentKey::LightClientBootstrap(_) => {
-                let key = key.content_id();
+            BeaconContentKey::LightClientBootstrap(content_key) => {
                 let is_key_available = self
-                    .lookup_content_key(key)
+                    .lookup_lc_bootstrap_block_root(&content_key.block_hash)
                     .map_err(|err| {
                         ContentStoreError::Database(format!(
-                            "Error looking up content key: {err:?}"
+                            "Error looking up lc bootstrap block root: {err:?}"
```

Comment:
```suggestion
                            "Error looking up light client bootstrap block root: {err:?}"
```

I don't think we should use short hands in error's